src/PortingAssistant.Client/PortingAssistantCLI.cs

```diff
@@ -118,6 +128,32 @@ public void HandleCommand(String[] args)
                     {
                         PortingProjects = o.PortingProjects.ToList();
                     }
+
+                    switch (o.MinimumLoggingLevel)
+                    {
+                        case "silent":
+                            MinimumLoggingLevel = Serilog.Events.LogEventLevel.Fatal + 1;
+                            break;
+                        case "fatal":
+                            MinimumLoggingLevel = Serilog.Events.LogEventLevel.Fatal;
+                            break;
+                        case "error":
+                            MinimumLoggingLevel = Serilog.Events.LogEventLevel.Error;
+                            break;
+                        case "warn":
+                            MinimumLoggingLevel = Serilog.Events.LogEventLevel.Warning;
+                            break;
+                        case "info":
+                            MinimumLoggingLevel = Serilog.Events.LogEventLevel.Information;
+                            break;
+                        case "debug":
+                            MinimumLoggingLevel = Serilog.Events.LogEventLevel.Debug;
+                            break;
+                        default:
+                            Console.WriteLine("Invalid logging level.");
+                            Environment.Exit(-1);
```

Comment:
Let's be opinionated here and set a default logging level (info?) if the input is invalid. In the print statement, inform the customer that we are setting a default logging level of X and also include the invalid input value.

Reply:
Fixed! Default level has been set to "debug" since that's what it was set to before the change, though all current logs are at level "info" or higher.